sneck/Deck.py

```diff
@@ -1018,6 +1110,16 @@ def get_id(self) -> int:
     def get_tag(self) -> str:
         return self.__tag
 
+    def set_title(self, title: str) -> None:
+        self.__title = title
+
+    def set_color(self, color: str) -> None:
+        self.__color = color
+
+    def set_deletion_time(self, deletion_time: Union[dt, int]) -> None:
+        self.__deletion_time = (dt.fromtimestamp(deletion_time).astimezone(tz.utc)
+                                if deletion_time != 0 else None)
+
```

Comment:
These setter functions must also make appropriate requests to update the corresponding values in the server. Unless I am missing something I don't think they currently do